include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -166,9 +239,21 @@ namespace SuperiorMySqlpp { namespace Converters
             }
         };
 
+        /**
+         * @class ToIntegerUnsignedImpl
```

Comment:
Using @class and @file is useless IMHO. It has no benefit for reading documentation directly in source code and this information is available automatically in generated documentation by Doxygen.